src/core/index.ts

```diff
@@ -135,11 +138,21 @@ export class CrowdinValidationError extends CrowdinError {
 /**
  * @internal
  */
-function handleError<T>(error: any = {}): T {
-    if (Array.isArray(error.errors)) {
+export function handleHttpClientError(error: HttpClientError): never {
+    const crowdinResponseErrors =
+        error instanceof AxiosError
+            ? error.response?.data?.errors
+            : error instanceof FetchClientJsonPayloadError
+            ? error.jsonPayload && typeof error.jsonPayload === 'object' && 'errors' in error.jsonPayload
+                ? error.jsonPayload.errors
+                : null
+            : null;
+
+    if (Array.isArray(crowdinResponseErrors)) {
         const validationCodes: { key: string; codes: string[] }[] = [];
         const validationMessages: string[] = [];
-        error.errors.forEach((e: any) => {
+        crowdinResponseErrors.forEach((e: any) => {
+            // @todo this is unsafe code. apply type guards
```

Comment:
What about this TODO? Could you please address this?

Reply:
removed. however, the remote crowdin errors payloard should be modeled. subjectively out-of-scope for this PR